### ISSUE TITLE
Phalcon\Config\Adapter\Ini parse error

### ISSUE THREAD

**Issue by zuozhehao** [phalcon/cphalcon#13604]
```
[task]
a.10[]  = 1
a.10[]  = 2
a.20[]  = 3
a.20[]  = 4

output:
Array
        (
            [a] => Array
                (
                    [10] => Array
                        (
                            [0] => 1
                            [1] => 2
                        )

                    [11] => Array
                        (
                            [0] => 3
                            [1] => 4
                        )

                )

        )
```

**Comment by CameronHall** [comment]
Hey @zuozhehao,

This looks to be related to https://github.com/phalcon/cphalcon/issues/12725. It _should_ be fixed in 4.0.0 :)

**Comment by sergeyklay** [comment]
Fixed in the `3.4.x` branch. Feel free to open a new issue if the problem appears again. Thank you for contributing.
